sdk/storage/storage-blob/src/BlobBatch.ts

```diff
@@ -481,9 +481,11 @@ class BatchRequestAssemblePolicyFactory implements RequestPolicyFactory {
 }
 
 class BatchHeaderFilterPolicy extends BaseRequestPolicy {
+  /* eslint-disable @typescript-eslint/no-useless-constructor*/
   constructor(nextPolicy: RequestPolicy, options: RequestPolicyOptions) {
     super(nextPolicy, options);
   }
+  /* eslint-enable @typescript-eslint/no-useless-constructor*/
```

Comment:
You can use `eslint-disable-next-line` above the constructor instead of using this form of disabling and enabling the rule. Same applies to other places where this format is used in this PR